src/pages/rewards/LaunchIncentivesPanel.tsx

```diff
@@ -153,7 +157,7 @@ const LaunchIncentivesContent = () => {
 const Styled: Record<string, AnyStyledComponent> = {};
 
 Styled.Panel = styled(Panel)`
-  background-color: var(--color-layer-4);
+  background-color: var(--color-layer-3);
```

Comment:
was this change reflected in designs as well?

Reply:
hm this page wasn't shown in designs, but when poking around, my interpretation of this was that `layer-4` was a bug since all the other panels are of color layer-3: 

| | Before (layer-4) | After (layer-3) |
| ----- | ----- | ----- |
| Classic | <img width="1064" alt="Screenshot 2024-02-01 at 6 28 57 PM" src="https://github.com/dydxprotocol/v4-web/assets/70078372/b4833e40-07c5-4297-8782-46fa6971a154"> | <img width="1072" alt="Screenshot 2024-02-01 at 6 27 32 PM" src="https://github.com/dydxprotocol/v4-web/assets/70078372/a715f004-177c-4237-a592-154fab496a47"> |
| Dark | <img width="1076" alt="Screenshot 2024-02-01 at 6 29 06 PM" src="https://github.com/dydxprotocol/v4-web/assets/70078372/c3c0f5e0-ae0f-41ed-9148-320d5aa9eb06"> | <img width="1087" alt="Screenshot 2024-02-01 at 6 27 38 PM" src="https://github.com/dydxprotocol/v4-web/assets/70078372/7f07bbda-27fd-4471-8a92-f6dbfdec5472"> |
| Light | <img width="1059" alt="Screenshot 2024-02-01 at 6 29 02 PM" src="https://github.com/dydxprotocol/v4-web/assets/70078372/a8d2adf9-7637-4906-9dec-e020179bb418"> | <img width="1088" alt="Screenshot 2024-02-01 at 6 27 12 PM" src="https://github.com/dydxprotocol/v4-web/assets/70078372/9c2ceb8f-5958-42e1-aea2-f96a23609fbb"> |

lmk if you think i'm wrong - but I feel pretty confident about this!

Reply:
Looks good to me!